lib/node_modules/@stdlib/lapack/base/dpttrf/README.md

```diff
@@ -0,0 +1,252 @@
+<!--
+
+@license Apache-2.0
+
+Copyright (c) 2024 The Stdlib Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+   http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+
+-->
+
+# dpttrf
+
+> Compute the `L * D * L ** T` factorization of real symmetric positive definite tridiagonal matrix `A`.
+
+<section class = "usage">
+
+## Usage
+
+```javascript
+var dpttrf = require( '@stdlib/lapack/base/dpttrf' );
+```
+
+#### dpttrf( N, D, E, info )
+
+Computes the `L * D * L ** T` factorization of real symmetric positive definite tridiagonal matrix `A`.
```

Comment:
```suggestion
Computes the `L * D * L**T` factorization of real symmetric positive definite tridiagonal matrix `A`.
```

Reply:
If you want to disambiguate, you can also use `L^T`, which is arguably easier to read and understand. We're not consistent about this, but my preference would be for using `^` to indicate transpose in docs.